client/db/src/lib.rs

```diff
@@ -1102,26 +1124,29 @@ impl<Block: BlockT> Backend<Block> {
 			}
 
 			let finalized = if operation.commit_state {
-				let mut changeset: sc_state_db::ChangeSet<Vec<u8>> = sc_state_db::ChangeSet::default();
+				let mut changesets = ChildrenMap::<sc_state_db::ChangeSet<Vec<u8>>>::default();
 				let mut ops: u64 = 0;
 				let mut bytes: u64 = 0;
```

Comment:
I think we should do: 
```suggestion
				let mut bytes = 0;
```

And just do the `bytes as u64` once when we call this `tally_writes`.